api-gateway/index.js

```diff
@@ -10,7 +10,10 @@ app.use(
     extended: false,
   })
 );
-
```

Comment:
Ovo mislim da ne treba. Slucajno sam puhovala . To je nesto sto sam probavala dok sam pokusavala da uradim

Reply:
Pa neka ga :) Taman da znamo sta se desava.